src/useList.tsx

```diff
@@ -18,7 +21,16 @@ export function useList<T extends any>(
     room.subscribe(l, next => {
       setList(next);
     });
+
+    local.subscribe(key, list => {
+      setList(list);
+    });
   }, [room, listName]);
 
-  return [list, setList];
+  function setAndBroadcastList(list: ListClient<T>) {
+    setList(list);
+    local.publish(key, list);
```

Comment:
does this publish also trigger the subscribe just above on line 25? And if so could that be an issue?

Reply:
Hmmm ya good point, we need a concept of an actor